src/test/scala/com/microsoft/hyperspace/index/rules/HyperspaceRuleSuite.scala

```diff
@@ -99,7 +103,7 @@ trait HyperspaceRuleSuite extends HyperspaceSuite {
       spark)
 
   def getIndexRootPath(indexName: String): Path =
-    new Path(systemPath, indexName)
+    PathUtils.makeAbsolute(new Path(systemPath, indexName), new Configuration)
```

Comment:
Is the change in this file required?

Reply:
yes without this system path could be relative